web/services/clusters.go

```diff
@@ -178,6 +180,63 @@ func (s *clustersService) GetAllTags() ([]string, error) {
 	return tags, nil
 }
 
+func (s *clustersService) GetAllClustersSettings() (models.ClustersSettings, error) {
+	var clusters []*entities.Cluster
+
+	err := s.db.
+		Preload("Hosts").
+		Find(&clusters).
+		Error
+
+	if err != nil {
+		return nil, err
+	}
+
+	clustersSettings := models.ClustersSettings{}
+	for _, cluster := range clusters {
+		var hosts []*models.ConnectionInfoAwareHost
+
+		selectedChecks, err := s.checksService.GetSelectedChecksById(cluster.ID)
+		if err != nil {
+			log.Error(err)
+			return clustersSettings, err
+		}
+
+		connectionSettings, err := s.checksService.GetConnectionSettingsById(cluster.ID)
+		if err != nil {
+			log.Error(err)
+			return clustersSettings, err
+		}
+
+		for _, host := range cluster.Hosts {
+			username := "root"
+
+			hostConnectionSettings, found := connectionSettings[host.Name]
+			if found {
+				username = hostConnectionSettings.User
+			}
+
+			if username == "" {
+				username = "cloudadmin" // load user from cloud info
```

Comment:
The default user being `cloudadmin` is an Azure-specific detail. Are we sure we can just hard-code this like that?

Reply:
It is temporary, that would be loaded from cloud metadata if the host happens to be on a cloud environement.
It'll be fixed soon. :wink: